src/stores/data.js

```diff
@@ -71,14 +76,18 @@ function getRow(dataInfo, key) {
 }
 
 export async function loadDatasets(nftsApi, allocations) {
-  const allDatasets = await getDatasets(nftsApi);
-  if (allDatasets.length === 0) {
+  let curRound = getEpoch().id;
+  //current round number is 0
+  const currentRoundDatasets = await getDatasets(nftsApi,0);
+  const lastRoundDatasets = await getDatasets(nftsApi,curRound-1)
+  if (currentRoundDatasets.length === 0) {
     datasets.set([]);
     return;
   }
   let newDatasets = [];
-  allDatasets.forEach((datasetInfo, key) => {
+  currentRoundDatasets.forEach((datasetInfo, key) => {
     datasetInfo.allocation = allocations.find((allocation) => allocation.nftAddress === datasetInfo.nft_addr)?.allocated/100 || 0
+    datasetInfo.lastRoundAPY = lastRoundDatasets.find((ld) => ld.nft_addr === datasetInfo.nft_addr)?.apy
```

Comment:
Perhaps this should be, 0, or NEW, or N/A.
This might not look too good when we start joining data.
```
5.31% / 3.1%
     3.45%
2.53% / 5.63%
```

Let's get there and then solve it.